### ISSUE TITLE
torch.complex32 missing in the stable torch release

### ISSUE THREAD

**Issue by Shangyint** [ise-uiuc/FreeFuzz#1]
Executing `cd src && python FreeFuzz.py --conf demo_torch.conf` yield the following error
```
  File "/home/shangyit/projects/FreeFuzz/src/classes/torch_api.py", line 14, in TorchArgument
    torch.complex32, torch.complex64, torch.complex128, torch.bool
  AttributeError: module 'torch' has no attribute 'complex32'
```
The torch developers did temporary disable this field in release 1.11.0 (newest stable). See [this](https://github.com/pytorch/pytorch/issues/72721).

**Comment by YangChenyuan** [comment]
Yes, `complex32` is disabled in the current version (1.11.0) of PyTorch due to https://github.com/pytorch/pytorch/issues/71680. You can delete `torch.complex32` at https://github.com/ise-uiuc/FreeFuzz/blob/881fc4eca1d5a8fa29f6980ba3d042d8e210ee93/src/classes/torch_api.py#L14
